lib/ask_export/targets/targets.rb

```diff
@@ -18,5 +18,9 @@ def self.find(name)
         target
       end
     end
+
+    def self.clear_cache
+      @target_cache = {}
```

Comment:
It feels odd that we're adding something to class that's only for use in the tests. I wonder if there's another way to do this in the tests?

Reply:
Yeah agree not ideal - but I'd spent amount time trying to figure out to fix this and couldn't see anything that didn't involve a huge code change. Do you think this is a blocking issue for this PR?

Reply:
No it's not a blocker. I spent a while looking for another solution on Friday as well and couldn't find one either, so I think it's find to go with this.